dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -90,16 +114,41 @@ void crash_signal_intercepted(int signal, const char *signal_name, const char *e
 
     // dump the log into a new file
     char filename[200];
-    snprintf(filename, 200, "%s/%llu", STORAGE_DIR, time_since_epoch());
+    snprintf(filename, 200, "%s/%llu", context.storage_dir, time_since_epoch());
```

Comment:
you can replace `200` by `sizeof(filename)`

Reply:
Nice one...I always thought that doing `sizeof(filename)` will give me the size of the pointer (4 bytes) 🤦